src/components/NcAppContent/NcAppContent.vue

```diff
@@ -268,6 +284,27 @@ export default {
 				},
 			}
 		},
+
+		realPageTitle() {
+			if (this.pageTitle) {
+				return t('{view} - {app} - {product}', { view: this.pageTitle, app: this.pageHeading || localizedAppName, product: productName })
+			}
+			if (this.pageHeading) {
+				return t('{view} - {app} - {product}', { view: this.pageHeading, app: localizedAppName, product: productName })
+			}
```

Comment:
These strings seem to not need translators.

It works fine when all the words are already translated, and in RTL, space and dash don't change the direction.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/fa75f9c4-3582-4b34-a2dc-17bea390f245)